test/Dapr.Client.Test/InvokeApiTest.cs

```diff
@@ -419,6 +420,47 @@ public async Task InvokeMethodAsync_WithReturnTypeAndData_WithNonDefaultVerb_Wit
             response.Name.Should().Be(invokedResponse.Name);
         }
 
+        [Fact]
+        public async Task InvokeMethodAsync_WithCancelledToken()
+        {
+            var client = new MockClient();
+            var data = new Response() { Name = "Look, I was invoked!" };
+            var invokeResponse = new InvokeResponse();
+            invokeResponse.Data = TypeConverters.ToAny(data);
+            
+            var response = 
+                client.Call<InvokeResponse>()
+                .SetResponse(invokeResponse)
+                .Build();
+
+            const string rpcExceptionMessage = "Call canceled by client";
+            const StatusCode rpcStatusCode = StatusCode.Cancelled;
+            const string rpcStatusDetail = "Call canceled";
```

Comment:
We might not want to put these errors messages in tests because they are coming from the Grpc library. Libraries are free to change their error messages 😆 

Suggestions for improvement:

- don't test the error message
- OR add our own error message where we throw OperationCancelledException and test that